source/api/content-state/0.9/index.md

```diff
@@ -409,23 +268,24 @@ In the following examples, the same Annotation is used each time. As JSON:
 
 An example of this usage would be a link from search results to a particular page of a digitized book, or a stored bookmark of a particular page (i.e., Canvas).
 
-Without the required encoding, the (invalid) link to the viewer would look like this:
+Without the required content-state-encoding, the (invalid) link to the viewer would look like this:
 
 ```html
 {% raw %}
-<a href='https://example.org/viewer?iiif-content={"type":"Annotation","motivation":"contentState","target":{"id":"http://dams.llgc.org.uk/iiif/2.0/4389767/canvas/4389772.json","type":"Canvas","partOf":[{"id":"http://dams.llgc.org.uk/iiif/2.0/4389767/manifest.json","type":"Manifest"}]}}'>INVALID, unencoded link to Viewer</a>
+<!-- INVALID, unencoded form -->
+<a href='https://example.org/viewer?iiif-content={"@context":"http://iiif.io/api/presentation/3/context.json","id": "https://example.org/content-states/1","type":"Annotation","motivation":"contentState","target":{"id":"http://dams.llgc.org.uk/iiif/2.0/4389767/canvas/4389772.json","type":"Canvas","partOf":[{"id":"http://dams.llgc.org.uk/iiif/2.0/4389767/manifest.json","type":"Manifest"}]}}'>INVALID, unencoded link to Viewer</a>
```

Comment:
For the example NLW manuscript could we use the https link so: https://damsssl.llgc.org.uk/iiif/2.0/4389767/manifest.json unfortunately it looks like this has difference canvas IDs.